WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCouponViewModel.swift

```diff
@@ -183,13 +192,16 @@ final class AddEditCouponViewModel: ObservableObject {
     @Published var categoryIDs: [Int64]
     @Published var showingCouponCreationSuccess: Bool = false
 
+    private var subscriptions: Set<AnyCancellable> = []
+
     /// Init method for coupon creation
     ///
     init(siteID: Int64,
          discountType: Coupon.DiscountType,
          stores: StoresManager = ServiceLocator.stores,
          storageManager: StorageManagerType = ServiceLocator.storageManager,
          currencySettings: CurrencySettings = ServiceLocator.currencySettings,
+         priceInputFormatter: PriceInputFormatter = PriceInputFormatter(),
```

Comment:
I may be missing something here, but why do we have to use `PriceInputFormatter` instead of `CouponAmountInputFormatter`?

Reply:
No reason at all, I started with the neutral formatter just for testing and forgot to switch to the dedicated Coupon Amount one, thanks for catching this one! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7036/commits/5e6a877a60cf362228c080dac8e4c258072bd956